components/motor/gpio/motor_encoder.go

```diff
@@ -399,17 +405,18 @@ func (m *EncodedMotor) rpmMonitorPass(pos, lastPos, now, lastTime int64, rpmDebu
 	// slow down so we don't overshoot
 	// halve and quarter rpm values based on seconds remaining in move
 
-	desiredRPM := m.state.desiredRPM
+	// desiredRPM := m.state.desiredRPM
```

Comment:
[nit] maybe remove unless you want to keep it for future reference since this is only part 1